supply-chain/config.toml

```diff
@@ -1007,31 +987,35 @@ version = "0.4.0"
 criteria = "safe-to-deploy"
 
 [[exemptions.windows-sys]]
-version = "0.36.1"
+version = "0.42.0"
+criteria = "safe-to-deploy"
+
+[[exemptions.windows_aarch64_gnullvm]]
+version = "0.42.0"
 criteria = "safe-to-deploy"
 
 [[exemptions.windows_aarch64_msvc]]
-version = "0.36.1"
+version = "0.42.0"
```

Comment:
We're trying to avoid any new exceptions nowadays, so could these new releases be vetted rather than updating existing exemptions?

Reply:
Done